lib/plugins/aws/info/display.js

```diff
@@ -172,6 +172,15 @@ module.exports = {
       legacy.consoleLog(message);
     }
 
+    if (isVerboseMode && this.gatheredData.outputs.length > 0) {
```

Comment:
nit: `this.gatheredData.outputs.length` is equivalent to `this.gatheredData.outputs.length > 0`